admin-manual/customization/authentication.rst

```diff
@@ -110,9 +114,234 @@ that are properly configured with an ``attribute_value`` and a valid
+LDAP server's directory tree. each attribute key/value pair should be entered
+separated by commas, with no spaces between them. 
+
+For example, if the target LDAP server has user accounts in an 
+Organizational Unit called "atomUsers" on a server with a domain of 
+``example.com``, then the Base DN setting in AtoM would be entered as: 
+
+.. code-block:: bash
+
+   ou=atomUsers,dc=example,dc=com 
+
+The **Bind Lookup Attribute** should be whatever attribute in your LDAP entries
+will be globally unique, unlikely to change, and used to identify individual 
+users. If you want the login experience to match AtoM's current expectations 
+(using an email address in the username field) and your LDAP server is configured
+to use **mail** attributes for every user, then you would enter ``mail`` into
+this field. 
+
+If ``mail`` is not used in your LDAP server, ``uid`` is a good option, as this
+typically shouldn't change even if the entry moves in the LDAP directory tree. 
+Barring that, ``cn`` could be used. Whatever is picked will define what type of
+information end users attempting to authenticate in AtoM should enter into the 
+login panel's "Email" field. 
+
+.. image:: images/ldap-config-example.*
+   :align: center
+   :width: 90%
+   :alt: An image of example valudes in the LDAP authentication settings
+
+In the above example, users would enter their LDAP username into AtoM's
+"Email" field in the login menu. If ``cn`` were used as the lookup attribute
+instead, then users would need to enter their LDAP common name.
+
+After configuring the LDAP settings
+-----------------------------------
+
+Once configured, users registered on the LDAP server should now be able to
+authenticate with their LDAP credentials to log into AtoM. 
+
+When a user authenticates for the first time, a new AtoM user is created in the 
+database. However, by default, this user is **not assigned to any user groups**,
+and will therefore only have the base permissions of the Authenticated user 
+group. An :term:`administrator` can modify the user permissions after the 
+new user has been created using AtoM's permissions module - for more information, 
+see: 
+
+* :ref:`edit-user-permissions`
+* :ref:`manage-user-accounts`
+
+.. IMPORTANT:: 
+
+   If switching to LDAP authentication *after* creating and configuring local
+   user accounts in AtoM, the most secure method to transition will likely be to 
+   delete the local user accounts and let the LDAP authentication create new 
+   ones, so that no passwords are stored in AtoM and no duplicate accounts exist. 
+
+   When AtoM attempts to authenticate an LDAP user, it will: 
+
+   * Fall back and check if a local account matches the user credentials if no
+     LDAP match is found. This can allow users with existing local accounts to 
+     bypass the LDAP authentication, and/or continue accessing AtoM even if the 
+     LDAP entry is removed.
+   * If a match IS found, AtoM will attempt to look for existing accounts by
```

Comment:
These two last bullet points do not continue from the lead-off "...authenticate an LDAP user, it will: If a match is found".